tools/arxiv/tests/quick_pipeline_test.py

```
@@ -0,0 +1,150 @@
+import json
+import time
+from pathlib import Path
+from datetime import datetime
+
+# Fix Python path
+script_dir = Path(__file__).resolve().parent
+project_root = script_dir.parent.parent.parent
+sys.path.insert(0, str(project_root))
+sys.path.insert(0, str(script_dir.parent))
+
+from pipelines.arxiv_pipeline import ACIDPhasedPipeline
```

Ruff (0.12.2)

Module level import not at top of file

(E402)